pkg/ostree/app.js

```diff
@@ -161,18 +166,49 @@
                     phantom_checkpoint();
                 });
 
-                $scope.knownVersions = function(os) {
-                    return client.known_versions_for(os);
+                $scope.switchOS = function(os) {
```

Comment:
This function is named as if it switches the operating system in OSTree. More likely it should be named ```displayOS``` or similar, since it only changes the display.